module/motion/Servos/src/Servos.hpp

```diff
@@ -60,11 +60,57 @@ namespace module::motion {
 
                     // Runs an emit for each servo
                     NUClear::util::unpack((emit<Task>(std::make_unique<Elements>(
-                                               group.servos[utility::motion::ServoMap<Elements>::value])),
+                                               group.servos.at(utility::motion::ServoMap<Elements>::value))),
                                            0)...);
                 });
         }
 
+        /// @brief Keeps track of which Group in the Sequence in add_sequence_provider should be emitted next
+        /// @tparam Sequence Used to have unique counts for each Sequence Provider.
+        template <typename Sequence>
+        struct Count {
+            long unsigned int count = 0;
```

Comment:
Should make a note somewhere that there is the potential for bugs here.
Really need a token system so we know which script we are getting the done events for.
Or do validation ourself that they are actually done